randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -988,10 +988,12 @@ Extra - room_id: [25]
   * L4 Hatch to Level 4 Security Room/Door to Security Access
   * Extra - door_idx: (51,)
   > By Hole
-      All of the following:
-          # TODO: underwater WJs?
-          Gravity Suit
-          Space Jump or Wall Jump (Intermediate)
+      Any of the following:
+          All of the following:
+              Gravity Suit
+              Space Jump or Wall Jump (Intermediate)
```

Comment:
I'm just noticing this now: shouldnt this be beginner walljump? at least with HJ? I dunno what exactly the consensus for WJ tricks is tbh...

Reply:
Added High Jump + Beginner Wall Jump option